src/NHibernate/Dialect/MySQL5Dialect.cs

```diff
@@ -1,4 +1,5 @@
 using System.Data;
+using NHibernate.Dialect.Function;
```

Comment:
There are no more changes in his file. This using is surely now undue.

But it is a bit strange that a Guid stored as a `binary(16)` succeeds the test with the base Dialect version of the function.